spacy_llm/backends/rest/registry.py

```diff
@@ -30,7 +32,7 @@ def backend_rest(
         Backend instance.
     """
 
-    backend = supported_apis[api](
+    backend = (supported_apis[api] if isinstance(api, str) else api)(
```

Comment:
I'd prefer making this more explicit for maintainability & readability purposes:
```suggestion
    backend_class = api
    if isinstance(api, str):
        backend_class = supported_apis[api]
    backend = backend_class(
```